README.md

```diff
@@ -0,0 +1,30 @@
+# A Jupyter notebook demonstrating the Bitcoin Test Framework #
+
```

Comment:
I think we should have a warning here. Something like:

Do not run test instances of bitcoind on the same machine that you store your Bitcoin private keys. This test framework shouldn't interfere with your standard bitcoind directory, but why risk it?